modules/cpmso-yaml-failure-domain-azure.adoc

```diff
@@ -8,17 +8,33 @@
 
 The control plane machine set concept of a failure domain is analogous to existing Azure concept of an link:https://learn.microsoft.com/en-us/azure/azure-web-pubsub/concept-availability-zones[_Azure availability zone_]. The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
 
-When configuring Azure failure domains in the control plane machine set, you must specify the availability zone name.
+When configuring Azure failure domains in the control plane machine set, you must specify the availability zone name. An Azure cluster uses a single subnet that spans multiple zones.
 
 .Sample Azure failure domain values
 [source,yaml]
 ----
-failureDomains:
-  azure: <1>
-  - zone: "1"
-  - zone: "2"
-  - zone: "3"
-  platform: Azure <2>
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+metadata:
+  name: cluster
+  namespace: openshift-machine-api
+spec:
+# ...
+  template:
+# ...
+    machines_v1beta1_machine_openshift_io:
+      failureDomains:
+        azure:
+        - zone: "1" # <1>
+        - zone: "2"
+        - zone: "3"
+        platform: Azure # <2>
+# ...
 ----
 <1> Each instance of `zone` specifies an Azure availability zone for a failure domain.
++
+[NOTE]
+====
+If the cluster is configured to use a single zone for all failure domains, this parameter is configured in the provider specification.
```

Comment:
Better?
```suggestion
If the cluster is configured to use a single zone for all failure domains, the `zone` parameter is configured in the provider specification instead of in the failure domain configuration.
```

Reply:
Yes, thanks!